src/components/SingleSelect.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import { SingleSelectField, SingleSelectOption } from '@dhis2/ui-core'
+
+import { normalizeProps } from './shared/helpers'
+import { fieldRenderProps } from './shared/propTypes'
+
+const createChangeHandler = props => ({ selected }) =>
```

Comment:
This is similar to the pattern used in the multiselect. I think to me it'd make more sense to pass it onChange directly, so it doesn't need to know about the shape of `props`. I assume that we'll take care of this later on instead of now, when we're looking at the normalizeProps function as well.

Reply:
I've added a reference to this comment in issue #153